### PR TITLE
NO-JIRA depend on sonar-javascript-plugin 7.4.4 for tests to check compatibility with 8.9 LTS

### PR DESCRIPTION
**Pull request by aurelien-poscia-sonarsource**

### CODE REVIEW

File: its/it-tests/src/test/java/com/sonar/scanner/api/it/ScannerApiTestSuite.java [hunk 1]
```diff
@@ -42,8 +42,8 @@ public class ScannerApiTestSuite {
   public static final Orchestrator ORCHESTRATOR = Orchestrator.builderEnv()
     .setSonarVersion(getSystemPropertyOrFail(SONAR_RUNTIME_VERSION))
     .useDefaultAdminCredentialsForBuilds(true)
-    // The scanner api should still be compatible with 7.9
-    .addPlugin(MavenLocation.of("org.sonarsource.javascript", "sonar-javascript-plugin", "7.0.1.14561"))
+    // The scanner api should still be compatible with 8.9
```
Comment (dbmeneses) [inline its/it-tests/src/test/java/com/sonar/scanner/api/it/ScannerApiTestSuite.java:45]: I find this comment misleading. The plugin may or may not be compatible with SonarQube and that is unrelated to the scanner-api.

**Review by dbmeneses — APPROVED** [review]
I

File: its/it-tests/src/test/java/com/sonar/scanner/api/it/ScannerApiTestSuite.java (hunk 1 above)
Comment (aurelien-poscia-sonarsource) [reply its/it-tests/src/test/java/com/sonar/scanner/api/it/ScannerApiTestSuite.java:45]: Ok so if I understand correctly, the plugin does not have to be compatible with "sonar-scanner-api". It just have to be compatible with SQ to not crash the IT as a side effect?

Comment (aurelien-poscia-sonarsource) [reply its/it-tests/src/test/java/com/sonar/scanner/api/it/ScannerApiTestSuite.java]: I'll merge with this new comment:
> We need to use a plugin compatible with both SonarQube DEV & SonarQube version defined in .cirrus.yml (currently SQ 7.9)

to fix the red build; let me know if you think it's not clear enough. And I'm still wondering whether it makes sense to test against SonarQube 7.9?

Comment (dbmeneses) [reply its/it-tests/src/test/java/com/sonar/scanner/api/it/ScannerApiTestSuite.java]: Sorry for the late reply. Yes, that's my understanding too. I don't have a strong opinion about testing against SQ 7.9. I think if it doesn't cause any problems, we could leave it, but I wouldn't oppose to remove it. The scanners are only guaranteed to be compatible with the LTS.
